.github/workflows/build_and_deploy.yml

```diff
@@ -6,16 +6,13 @@
 # commit to `main` occurs in `napari/napari`
 
 name: Build & Deploy PR Docs
-
+# this does a full build of the docs for deployment
```

Comment:
In addition to or instead of the comment, it would be helpfult to change the action `name: Build full docs and deploy`